### PR TITLE
#41: removed the deleted properties usage

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: src/main/java/com/exasol/adapter/jdbc/BaseConnectionDefinitionBuilder.java [hunk 1]
```diff
@@ -32,44 +21,11 @@
...
-        } else if (properties.hasConnectionName()) {
-            return getNamedConnection(properties);
+        if (properties.hasConnectionName()) {
+            return "AT " + properties.getConnectionName();
         } else {
-            throw new IllegalArgumentException(MISSING_CONNECTION_DETAILS_ERROR);
+            throw new IllegalArgumentException(
+                    "Please, provide a mandatory property \"" + CONNECTION_NAME_PROPERTY + "\".");
         }
     }
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/jdbc/BaseConnectionDefinitionBuilder.java:30]: So happy this code is now gone! :-) :wave:

File: src/main/java/com/exasol/adapter/jdbc/RemoteConnectionException.java [hunk 2]
```diff
@@ -15,4 +15,13 @@
     public RemoteConnectionException(final String message, final Throwable cause) {
         super(message, cause);
     }
+
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/jdbc/RemoteConnectionException.java:18]: If you update a serializable class, please regenerate the UID!

File: src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java [hunk 3]
```diff
@@ -36,33 +41,33 @@ void beforeEach() {
     }
 
     @Test
-    void testPushdown() throws AdapterException {
+    void testPushdown() throws AdapterException, ExaConnectionAccessException {
         final PushDownResponse response = pushStatementDown(TestSqlStatementFactory.createSelectOneFromSysDummy());
         assertThat(response.getPushDownSql(), equalTo("IMPORT INTO (c1 DECIMAL(10, 0))" //
                 + " FROM JDBC" //
-                + " AT 'jdbc:derby:memory:test;create=true;' USER '' IDENTIFIED BY ''"//
+                + " AT derby_connection"//
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java:48]: Use upper case please.

File: src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java [hunk 4]
```diff
@@ -36,33 +41,33 @@ void beforeEach() {
...
+            throws AdapterException, ExaConnectionAccessException {
         setTestSqlDialectProperty();
-        setDerbyConnectionProperties();
+        setDerbyConnectionNameProperty();
         this.rawProperties.put(SCHEMA_NAME_PROPERTY, "SYSIBM");
         final List<TableMetadata> involvedTablesMetadata = null;
         final PushDownRequest request = new PushDownRequest(TEST_DIALECT_NAME, createSchemaMetadataInfo(), statement,
                 involvedTablesMetadata);
         final ExaMetadata exaMetadataMock = mock(ExaMetadata.class);
+        when(exaMetadataMock.getConnection("derby_connection")).thenReturn(EXA_CONNECTION_INFORMATION);
```
Comment (redcatbear) [inline src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java:61]: dto.

And in all other occurrences.

File: src/main/java/com/exasol/adapter/jdbc/RemoteConnectionException.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/jdbc/RemoteConnectionException.java:18]: done

File: src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java (hunk 3 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java:48]: fixed

File: src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java (hunk 4 above)
Comment (AnastasiiaSergienko) [reply src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java:61]: done

**Pull request merged**